components/automate-ui/src/app/modules/token/token-list/api-token-list.component.spec.ts

```diff
@@ -26,7 +28,9 @@ describe('ApiTokenListComponent', () => {
         StoreModule.forRoot({
           apiTokens: apiTokenEntityReducer,
           policies: policyEntityReducer,
-          projectsFilter: projectsFilterReducer
+          projectsFilter: projectsFilterReducer,
+          notifications: notificationEntityReducer, // not used here but needed to suppress warnings
+          clientRunsEntity: clientRunsEntityReducer // not used here but needed to suppress warnings
```

Comment:
The specific cause of all the console warnings was that these two properties were missing from the state in 14 test files. This fix was used in the first 10 files.